test-cases/performance/perf-regression-mixed-cdc.yaml

```diff
@@ -0,0 +1,29 @@
+test_duration: 300
+
+stress_cmd_w: "cassandra-stress write no-warmup cl=QUORUM duration=60m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=100 -pop seq=1..3000000"
+
+n_db_nodes: 3
+n_loaders: 4
+n_monitor_nodes: 1
+
+instance_type_loader: 'c4.2xlarge'
+instance_type_monitor: 't3.small'
+instance_type_db: 'i3.4xlarge'
```

Comment:
Change to i3.2xlarge so it will be comparable to regular test_write, test_read.

You'll need to clean all results you have on elasticsearch with the i3.4xlarge.

Reply:
Changed